### DISCUSSION TITLE
NGINX Proxy Manager (I think....)

### DISCUSSION THREAD

**Discussion by NorthantsAndy** [tteck/Proxmox#2452] [Help Needed]
I use LXCs for PiHole, dynamic ip DNS and NGINX PM to be able to forward domains to my internal systems.

My domains are whitelisted in PH, working with the dynamic ip client and are online in NPM.

I can ping everything internally and externally, but when you enter the URLs you get nowt - not even to redirection I have set up .

Any ideas?

**Comment by NorthantsAndy** [comment]
Bump bump bumpety bump...

**Comment by Mellowlynx** [comment]
This is not a problem with the scrip made by ttech, but probably a domain/DNS error.
So set up the DNS of the domain to point to your wan IP (use Cloudflare Proxy to hide your IP), or set the Local DNS Records in PiHole correctly.
Else, ask a tech friend, google, YouTube, or ChatGPT for help.
